src/cloud/release-notes/backward-incompatible-changes.md

```diff
@@ -0,0 +1,112 @@
+`vendor/bin/ece-tools docker:config:convert` | `vendor/bin/ece-docker  image:generate:php`
+
+In previous {{ site.data.var.ct }} releases, you could use the `m2-ece-build` and `m2-ece-deploy` commands to configure deployment hooks in the `.magento.app.yaml` file. When you update to v2002.1.0, check the `hooks` configuration in the `.magento.app.yaml` file for the obsolete commands, and replace them if needed.
+
+## Magento Cloud Patches changes
+
+-  **Remove downloaded patches**–The `{{site.data.var.mcp}}` package bundles all patches available from the [Magento Technical resources](https://magento.com/tech-resources/download) page and applies them automatically when you deploy to the Cloud.  To prevent patch conflicts after upgrading to {{site.data.var.ct}} 2002.1.0 or later, remove any Magento-supplied patches that you downloaded and added to your project manually.
+
+-  **Updating the apply patches command**–We moved the command for applying patches to Magento Cloud from the `vendor/bin/ece-tools` directory to the `vendor/bin/ece-patches` directory. If you use this command to apply patches manually, use the new path.
+
+   > Manually apply patches
+
+   ```bash
+   php ./vendor/bin/ece-patches apply
+   ```
+
+## Magento Cloud Docker changes
+
+-  **The minimum PHP version requirement is now PHP 7.1**–If your Magento Cloud Docker host is running an earlier version, upgrade to PHP v7.1 or later.
+
+-  **Magento Cloud Docker command changes**–
+
+   -  **Updating Magento Cloud Docker commands for Docker build operations**–We moved the Magento Cloud Docker commands from the `vendor/bin/ece-tools` directory to the `vendor/bin/ece-docker` directory. Update your scripts and commands to use the new path.
+
+      After upgrading to the {{site.data.var.ct}} 2002.1.0 or {{site.data.var.mcd}} 1.0.0, use the following command to view available `ece-docker` commands.
+
+      ```bash
+      php ./vendor/bin/ece-docker list
+      ```
+
+   -  **Updating the Magento Cloud docker-compose commands**–We changed the name of the `./bin/docker` directory to `./bin/magento-docker`.  Update your scripts and commands to use the new path.
+
+   -  **Using temporary containers**–In previous versions, the containers created by `bin/magento-docker` command operations were not removed, so you could use them for other operations.  Now, the `magento-docker` commands remove any containers they create after the command completes.
+
+      If you want to keep a container created by a docker-compose operation, use the `docker-compose run` command instead of the `bin/magento-docker` command.
+
+   -  **Running post-deploy hooks**–The `cloud-deploy` command no longer runs post deploy hooks. You must use the new `cloud-post-deploy` command to run post deploy hooks after you deploy.
+
+      ```bash
+      bin/magento-docker ece-deploy
+      ```
+
+      ```bash
+      bin/magento-docker ece-post-deploy
+      ```
```

Comment:
These should be in the same code block. This is an example of how the commands should be run one after another in a script.